src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -453,12 +453,15 @@ let add_breadcrumb_exn t breadcrumb =
     Mina_block.Validated.valid_commands
     @@ Breadcrumb.validated_transition breadcrumb
   in
+  let tx_hash_json =
+    Fn.compose
+      Mina_transaction.Transaction_hash.(Fn.compose to_yojson hash_command)
+      User_command.forget_check
+  in
   [%str_log' trace t.logger] Added_breadcrumb_user_commands
     ~metadata:
       [ ( "user_commands"
-        , `List
-            (List.map user_cmds
-               ~f:(With_status.to_yojson User_command.Valid.to_yojson) ) )
+        , `List (List.map user_cmds ~f:(With_status.to_yojson tx_hash_json)) )
```

Comment:
It is this change that's causing the [verification key test](https://buildkite.com/o-1-labs-2/mina-o-1-labs/builds/30320#018a754c-b7ca-400c-9574-787d711837a8) to fail. You'll want to find using transaction hash [here](https://github.com/MinaProtocol/mina/blob/fccb57dcb2ed38e97245c3fdb770467af76df6c3/src/lib/integration_test_lib/wait_condition.ml#L294)

Reply:
@deepthiskumar I believe I do now exactly as you say: https://github.com/MinaProtocol/mina/pull/14054/files#diff-76bbbeef12ee2aa92308935beebc5d6b45b2343aeb90e561b3e3cbe76e7aa045R289

Also, awkwardly, although nearly all intg tests and other sections of verify test rely on the same `zkapp_to_be_included_in_frontier` function, only last two sections of verification-key test fail.